docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+
+SQL::
+* Add text formatting support for multivalue {es-pull}68606[#68606]
+* Add xDBC and CLI support. QA CSV specs {es-pull}68966[#68966]
+* Export array values through result sets {es-pull}69512[#69512]
+* Improve alias resolution in sub-queries {es-pull}67216[#67216] (issue: {es-issue}56713[#56713])
+* Improve the optimization of null conditionals {es-pull}71192[#71192]
+* Push `WHERE` clause inside subqueries {es-pull}71362[#71362]
+* Use Java `String` methods for `LTRIM/RTRIM` {es-pull}57594[#57594]
+* QL: Make canonical form take into account children {es-pull}71266[#71266]
+* QL: Polish optimizer expression rule declaration {es-pull}71396[#71396]
+* QL: Propagate nullability constraints across conjunctions {es-pull}71187[#71187] (issue: {es-issue}70683[#70683])
+
+Search::
+* Completely disallow setting negative size in search {es-pull}70209[#70209] (issue: {es-issue}69548[#69548])
+* Make `0` as invalid value for `min_children` in `has_child` query {es-pull}41347[#41347]
+* Return error when remote indices are locally resolved {es-pull}74556[#74556] (issue: {es-issue}26247[#26247])
+* [REST API Compatibility] Nested path and filter sort options {es-pull}76022[#76022] (issues: {es-issue}42809[#42809], {es-issue}51816[#51816])
+* [REST API Compatibility] `CommonTermsQuery` and `cutoff_frequency` parameter {es-pull}75896[#75896] (issues: {es-issue}42654[#42654], {es-issue}51816[#51816])
+* [REST API Compatibility] Allow first empty line for `_msearch` {es-pull}75886[#75886] (issues: {es-issue}41011[#41011], {es-issue}51816[#51816])
+* Node level can match action {es-pull}78765[#78765]
+* TSDB: Add time series information to field caps {es-pull}78790[#78790] (issue: {es-issue}74660[#74660])
+* Add new kNN search endpoint {es-pull}79013[#79013] (issue: {es-issue}78473[#78473])
+* Disallow kNN searches on nested vector fields {es-pull}79403[#79403] (issue: {es-issue}78473[#78473])
+* Ensure kNN search respects authorization {es-pull}79693[#79693] (issue: {es-issue}78473[#78473])
+* Load kNN vectors format with mmapfs {es-pull}78724[#78724] (issue: {es-issue}78473[#78473])
+* Support cosine similarity in kNN search {es-pull}79500[#79500]
+* Node level can match action {es-pull}78765[#78765]
+* Check nested fields earlier in kNN search {es-pull}80516[#80516] (issue: {es-issue}78473[#78473])
+
+Security::
+* Add a tool for creating enrollment tokens {es-pull}74890[#74890]
+* Add the Enroll Kibana API {es-pull}72207[#72207]
+* Change default hashing algorithm for FIPS 140 {es-pull}55544[#55544]
+* Create enrollment token {es-pull}73573[#73573] (issues: {es-issue}71438[#71438], {es-issue}72129[#72129])
+* Enroll node API {es-pull}72129[#72129]
+* Not encoding the Api Key in Enrollment token {es-pull}74510[#74510] (issue: {es-issue}73573[#73573])
+* Configure security for the initial node CLI {es-pull}74868[#74868]
+* Adding base `RestHandler` class for Enrollment APIs {es-pull}76564[#76564] (issue: {es-issue}76097[#76097])
```

Comment:
```suggestion
```

Enhancement on unreleased code

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c.